pkg/hs/urbit-king/lib/Urbit/Arvo/Common.hs

```diff
@@ -1,3 +1,5 @@
+{-# LANGUAGE StrictData #-}
```

Comment:
I have mixed feelings about this. Basically, it makes readability worse (you have to remember that the meaning of a data definition in these files is different), but writability better (basically no chance of accidentally introducing laziness when you add a new ev/ef). It's not clear to me what the right call is. For what it's worth, I've only ever seen this used as a project-wide choice. But I'll sign off on it for now.